pkg/tests/integration/test_salt_user.py

```diff
@@ -0,0 +1,19 @@
+import psutil
+import pytest
+
+pytestmark = [
+    pytest.mark.skip_on_windows,
+]
+
+
+def test_salt_user_master(salt_master):
+    """
+    Test the correct user is running the Salt Master
+    """
+    user = "salt"
+    match = False
+    for _proc in psutil.process_iter(["username", "cmdline", "name"]):
```

Comment:
I believe the `salt_master` object has a `.pid` attribute.
Get that from psutil and itereate through children of need be?

Reply:
do you mean ppid? To get the parent process? When I do that it tries to get the process of the test runner. Are you okay with my previous approach?

Reply:
https://github.com/saltstack/pytest-shell-utilities/blob/main/src/pytestshellutils/shell.py#L479
so, `psutil.Process(salt_master.pid)`

Your approach is looking for names which is fragile. But I'm not against it.